scripts/start-publisher.sh

```diff
@@ -48,4 +48,5 @@ cargo run -p fuel-streams-publisher -- \
     --relayer-v2-listening-contracts $RELAYER_V2_LISTENING_CONTRACTS \
     --relayer-da-deploy-height $RELAYER_DA_DEPLOY_HEIGHT \
     --relayer-log-page-size $RELAYER_LOG_PAGE_SIZE \
-    --reserved-nodes /dns4/p2p-testnet.fuel.network/tcp/30333/p2p/16Uiu2HAmDxoChB7AheKNvCVpD4PHJwuDGn8rifMBEHmEynGHvHrf
+    --reserved-nodes /dns4/p2p-testnet.fuel.network/tcp/30333/p2p/16Uiu2HAmDxoChB7AheKNvCVpD4PHJwuDGn8rifMBEHmEynGHvHrf \
+    --server-addr 0.0.0.0:9000
```

Comment:
Do we want to update our `Dockerfile` as well?

Reply:
We have to , yes. Now we are running an http server with metircs and healthchecks.